docs/en/data_types/enum.md

```diff
@@ -1,13 +1,13 @@
 # Enum
 
-Enumerated type storing pairs of the `'string' = integer` format.
+Enumerated type storing pairs in `'string' = integer` format.
```

Comment:
this sounds confusing since real data is stored only as integers, I'd suggest rewriting it without using "store"